### PR TITLE
Adding the default animation values & mixin to be used within components

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add a settings file with the default animation values and a mixin to use them.

## QA
They are currently not used by the framework but are available to start testing. You could QA by adding `@include animation();` to the button element and it should take the default styling. 

## Details
Fixes https://github.com/ubuntudesign/vanilla-framework/issues/796

### CODE REVIEW

**Comment by anthonydillon** [comment]
@barrymcgee good shout. Done.

**Comment by barrymcgee** [comment]
@anthonydillon Spell checker bustin' your..

File: docs/settings/_animation-settings.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+collection: settings
+title: Animations
+---
+
+Vanilla has a cross matrix of duration and easing to apply animation across
+it required components. Components that animate staticly such as, the
```
Comment (barrymcgee) [inline docs/settings/_animation-settings.md]: 'staticly' should be 'statically'

Comment (barrymcgee) [inline docs/settings/_animation-settings.md]: Remove the comma ^

File: docs/settings/_animation-settings.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+background of buttons. Have a much shorter duration then components with
```
Comment (barrymcgee) [inline docs/settings/_animation-settings.md]: Remove the full stop ^^

File: docs/settings/_animation-settings.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Vanilla's easing consists of `easeInCubic` and `easeOutCubic`.
+
+Setting  | Default value
+ ------------- | -------------
+`out`   | `cubic-bezier(.215, .61, .355, 1)`
+`in`   | `cubic-bezier(.55, .055, .675, .19)`
+
+### Usage
+
+Animations are preset to components. Components can we modified or extended to
```
Comment (barrymcgee) [inline docs/settings/_animation-settings.md]: 'can we' should be 'can be'

File: docs/settings/_animation-settings.md (hunk 1 above)
Comment (anthonydillon) [inline docs/settings/_animation-settings.md]: Nice, its working!

**Comment by anthonydillon** [comment]
Fixed up the typos in the documentation and also fixed a lint warning as a drive by.

**Pull request merged**